docs/tutorials/creating-a-runtime-module.md

```diff
@@ -2,46 +2,68 @@
 title: "Creating a Runtime Module"
 ---
 
-In this tutorial, you'll write a Substrate runtime module that lives in its own crate, and include it in a node based on the node-template.
+In this tutorial, you'll write a Substrate runtime module that lives in its own
+crate, and include it in a node based on the node-template.
 
 ## Setup Your Development Environment
 
-If you haven't already, follow the [fast installation](getting-started/installing-substrate.md#fast-installation) for getting started on Substrate. It will provide you with all the development resources you need to build Substrate and the [`substrate-up` scripts](getting-started/using-the-substrate-scripts) for quickly setting up new nodes.
+If you haven't already, follow the
+[fast installation](getting-started/installing-substrate#fast-installation) for
+getting started on Substrate. It will provide you with all the development
+resources you need to build Substrate and the
+[`substrate-up` scripts](getting-started/using-the-substrate-scripts) for
+quickly setting up new nodes.
 
-## Create your Node
+### Cloning Node and Module Template to Start
 
-We'll begin by creating a brand new node template. If you've followed other runtime tutorials, you've probably started this way before:
-
-```bash
-substrate-node-new modular-chain <your name>
-```
-
-In this tutorial, we're not going to write our module directly as part of the node template, but rather as a separate Rust crate. This approach provides a few advantages:
+We're not going to write our module directly as part of the node template, but
+rather as a separate Rust crate. This approach provides a few advantages:
 
 1. Our module can be easily imported into other nodes in the future.
-2. Our module can be published to places like GitHub or Crates.io without publishing our entire node in the same place.
+2. Our module can be published to places like GitHub or Crates.io without
+publishing our entire node in the same place.
 
-However, we do still need to set up a Substrate node to actually run our module.
+However, to effectively develop and test with your own runtime module, you need
+to first setup a single-node Substrate blockchain. Then inside the node runtime
+add in your own additional module.
 
-## Clone the Module Template Repository
+We'll begin from cloning the necessary repositories:
 
-So if our module isn't going to live in the node template, where _is_ it going to live?
+```bash
+cd my-playground-folder
```

Comment:
this is a really small nitpick, but I think we should stick to simple folder names like: `example`

I could literally imagine the pain of users who have to type this in command line.

If it will be a "longer" name, then might as well be a descriptive one?

`runtime-module-tutorial`

Reply:
Do you also need a `mkdir` here, or you assume the user knows to do that?